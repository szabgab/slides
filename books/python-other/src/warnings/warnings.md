# Warnings

* warn

{% embed include file="src/examples/warnings/warnings.py" %}



