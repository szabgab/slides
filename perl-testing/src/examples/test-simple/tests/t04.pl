use strict;
use warnings;

use FindBin qw($Bin);
use lib "$Bin/../lib";
use MySimpleCalc qw(sum);

print sum(1, 1), "  2\n";
print sum(2, 2), "  4\n";
print sum(2, 2, 2), "  6\n";
print sum(3, 3), "  6\n";
print sum(4, 4), "  8\n";
print sum(5, 5), "  10\n";
print sum(6, 6), "  12\n";
print sum(7, 7), "  14\n";

