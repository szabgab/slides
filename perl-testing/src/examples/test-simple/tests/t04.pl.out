2  2
4  4
4  6
6  6
8  8
10  10
12  12
14  14
